pkg/autoscaler/autoscaler.go

```diff
@@ -161,10 +201,10 @@ func (a *Autoscaler) Scale(ctx context.Context, now time.Time) (int32, bool) {
 	defer a.statsMutex.Unlock()
 
 	// 60 second window
-	stableData := newTotalAggregation()
+	stableData := newTotalAggregation(60 * time.Second)
 
 	// 6 second window
-	panicData := newTotalAggregation()
+	panicData := newTotalAggregation(6 * time.Second)
```

Comment:
Should these be `config.StableWindow`/`config.PanicWindow` respectively?

Reply:
Ah yes.  Done.